src/sentry/shared_integrations/client.py

```diff
@@ -293,3 +295,15 @@ def post(self, *args, **kwargs):
 
     def put(self, *args, **kwargs):
         return self.request("PUT", *args, **kwargs)
+
+    def head(self, *args, **kwargs):
```

Comment:
I added this in case future people want to do a non-cached HEAD request but idk how necessary it is to leave in

Reply:
@ceorourke why not have `head_cached` call the `head` function?

Reply:
@scefali I could do that, I was copying what `get_cached` is doing.

Reply:
@scefali okay 👀  it now